src/main/kotlin/org/objectionary/ddr/sources/Sources.kt

```diff
@@ -37,26 +37,26 @@ class SrsTransformed(
 ) : Sources {
     private val logger = LoggerFactory.getLogger(this.javaClass.name)
 
-    /** @property path path to directory with source files or single source file */
-    val path: Path = Path.of(strPath)
+    /** @property inPath path to directory with source files or single source file */
+    val inPath: Path = Path.of(strPath)
```

Comment:
@c71n93 We are trying to avoid compound names. According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
@Graur I'm planning to add public field `resPath` with path to result directory in the next PR. Thus, compound naming is necessary to distinguish between input and result paths. I can name these fields `inputs` and `results` and in context of this class it will be fine. But these fields are public, so in another context it won't be clear that `sources.results` is the path to the results directory. What do you think?

Reply:
@c71n93 Maybe just `input` and `path`? It is just a single path to directory, am I right?